src/host/CursorBlinker.cpp

```diff
@@ -218,7 +218,7 @@ void CursorBlinker::KillCaretTimer()
         // A failure to delete the timer with the LastError being ERROR_IO_PENDING means that the timer is
         // currently in use and will get cleaned up when released. Delete should not be called again.
         // We treat that case as a success.
-        if (bRet == false && GetLastError() != ERROR_IO_PENDING)
+        if (!bRet && GetLastError() != ERROR_IO_PENDING)

```

Comment:
I appreciate your excitement! Would you please try to contain stylistic changes to one pull request per _type of change_? This is the `NULL`->`nullptr` pull request, not the "check boolean conditions differently" pull request.

Thanks!